src/javascript/binary/websocket_pages/user/ico_subscribe.js

```diff
@@ -43,6 +43,7 @@ const ICOSubscribe = (() => {
         BinarySocket.wait('website_status').then((response) => {
             if (response.website_status.ico_status === 'closed') {
                 $(form_id).replaceWith($('<p/>', { class: 'notice-msg center-text', text: localize('The ICO is currently unavailable.') }));
+                ICOcountDown();
```

Comment:
Does this display the countdown even after the ICO ended? (`ico_status` is `closed` also after it ends)

Reply:
@binary-hamedanchi No, it doesn't. I'm checking it [here](https://github.com/binary-com/binary-static/pull/3016/files#diff-0086c60d4479798647265119fb303f63R118) if there's some time left or not.

Reply:
I see. Then it needs to make it invisible again, in case the page was still open.

Reply:
I think my earlier comment got removed, adding it here.

I'm handling the case where user is still on the page and timer runs down by doing a page reload when the time_left is 0 seconds.